roles/openshift_health_checker/action_plugins/openshift_health_check.py

```diff
@@ -69,13 +69,15 @@ def run(self, tmp=None, task_vars=None):
                         msg=str(e),
                     )
 
+            if check.changed:
+                r["changed"] = True
```

Comment:
Yes, but I thought it wouldn't hurt to allow the check to continue to return it in the result hash if desired for some reason. But thinking about it, why complicate what is really quite simple... "one way to do it" is probably best here.